ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -1347,6 +1383,81 @@ class PackedBubbleSeries extends BubbleSeries implements Highcharts.DragNodesSer
             null;
     }
 
+
+    /**
+     * Set the state of the series. Called internally on mouse interaction
+     * operations, but it can also be called directly to visually
+     * highlight a series.
+     *
+     * @function Highcharts.Series#setState
+     *
+     * @param {Highcharts.SeriesStateValue|""} [state]
+     *        The new state, can be either `'hover'`, `'inactive'`, `'select'`,
+     *        or `''` (an empty string), `'normal'` or `undefined` to set to
+     *        normal state.
+     * @param {boolean} [inherit]
+     *        Determines if state should be inherited by points too.
+     */
+    public setState(
+        state?: (StatesOptionsKey|''),
+        inherit?: boolean
+    ): void {
+
+
+        // Run default method
+        Series.prototype.setState.apply(this, arguments);
+
+        let series = this,
+            options = series.options,
+            stateOptions = options.states,
+            graph = series.graph,
+            layoutOptions = options.layoutAlgorithm,
+            lineWidth,
+            attribs,
+            stateAnimation = pick(
+                (
+                    (stateOptions as any)[state || 'normal'] &&
```

Comment:
Do we need `as any` in here? Maybe we need to import something from StateOptions file